README.md

```diff
@@ -2,33 +2,35 @@
 # How to Update the [MHKiT Website](https://MHKiT-Software.github.io/MHKiT/)
 
 ## Download/Install Required Packages
-### Step 1. Download and Install Python 
+### Step 1. Download and Install Python
   - Windows: Download and install Python using [Anaconda](https://www.anaconda.com/distribution/)
   - **NOTE:** MAC/LINUX skip to Step 3, Python is already installed and added to path
- 
+
 ### Step 2. Add Python to System Path
-  - Windows: Modify the PATH in environmental variables to include: C:\PYTHON;C:\PYTHON\Scripts 
+  - Windows: Modify the PATH in environmental variables to include: C:\PYTHON;C:\PYTHON\Scripts
   ([more info here](http://stackoverflow.com/questions/3701646/how-to-add-to-the-pythonpath-in-windows-7))
-  - **NOTE:** Anaconda skip to Step 3,  Python is already added to path 
+  - **NOTE:** Anaconda skip to Step 3,  Python is already added to path
   - **NOTE:** MAC/LINUX skip to Step 3, Python is already added to path
-  
+
 ### Step 3. Download/Install [Sphinx](http://www.sphinx-doc.org/en/stable/index.html) package
-  - Install Sphinx from cmd ``pip install -U Sphinx`` 
+  - Install Sphinx from cmd ``pip install -U Sphinx``
     ([more info here](http://www.sphinx-doc.org/en/master/usage/installation.html))
   - **NOTE:** You may need to add PROXY settings ([see info here](https://cinhtau.net/2018/04/16/python-proxy-windows/))
 
 ### Step 4. Download/Install Sphinx extensions
```

Comment:
let's move the contents of step 4 into the previous steps. E.g. include the Sphinx extensions in the install Sphinx step